workflow-templates/im-build-db-ci.yml

```diff
@@ -94,10 +94,8 @@ jobs:
       # They will build your database on the Action Runner and run any tests you have.
       # The steps that come next aren't going to be needed by every repo, so feel free to remove what is unnecessary for yours.
       #########################################################################################################################
-
```

Comment:
I'll bet all of these space removals between comments are the yaml extension's fault. I had that problem myself. It's unfortunate because the space really helps with readability IMO.

Reply:
You have to comment out the empty line to keep it.

Reply:
I think this is a good reason to get everyone on the same extensions.  Cause if one person has it and others don't (like when Roy had it and no one else did), everything looks like it has major changes when an extension-haver saves the file. :)

Reply:
Do we want to compile a list somewhere? It might be good to share with teams as we transition them as well.

Reply:
Yep, it's already in the spreadsheet.  I just have the two extensions for YAML and GitHUb Actions.